apps/system/test/marionette/browser_chrome_new_window_test.js

```diff
@@ -35,27 +35,26 @@ marionette('Browser Chrome - Open New Window', function() {
     home.waitForLaunch();
   });
 
-  function newWindow() {
-    // Open the context menu and click open new window
-    client.switchToFrame();
-    system.appChromeContextLink.click();
-    assert.ok(system.appChromeContextMenu.displayed());
-
-    var newWindowLink = system.appChromeContextMenuNewWindow;
-    assert.ok(newWindowLink.displayed());
-    newWindowLink.click();
-  }
-
   test('open new window', function() {
     // Use the home-screen search box to open up the system browser
     var url = server.url('sample.html');
     rocketbar.homescreenFocus();
     rocketbar.enterText(url, true);
 
-    // Count the number of currently open apps
+    // Wait for all app windows to show up
+    client.helper.wait(1000);
```

Comment:
This kinda thing always makes me nervous. We need to wait on specific things happening, not an arbitrary amount of time. Is there nothing specifically we can check for in the DOM?